package.json

```diff
@@ -5,7 +5,8 @@
     "packages/*"
   ],
   "scripts": {
-    "start": "openmrs develop --sources 'packages/esm-*-app/'",
+    "start": "openmrs develop --sources packages/esm-patient-chart-app/ --sources",
```

Comment:
Drop the second `--sources`. If somebody really wants to use that it can / should be appended explicitly, too.